README.md

```diff
@@ -1,34 +1,28 @@
-# acmehealth-server
-Node Server for CRUD API Management Operations
+# AcmeHealth Server
+
+The projects serves as the backend for the AcmeHealth system. It performs simple OAuth token validation and CRUD operations for appointements. After configuring this project, you must also deploy one of the front end applications.
+
+The front end projects are available in [Angular single page application](https://github.com/jmelberg/acmehealth-spa), [Djagno framework application](https://github.com/oktadeveloper/okta-acmehealth-client-django), and a [Swift mobile app](https://github.com/jmelberg/acmehealth-swift).
```

Comment:
Actually, these samples do not handle the same use case. They are only for starting the login-flow and do not handle scheduling/ requests.